server/apiv2/handlers/tso_keyspace_group.go

```diff
@@ -0,0 +1,137 @@
+// Copyright 2023 TiKV Project Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package handlers
+
+import (
+	"net/http"
+	"strconv"
+
+	"github.com/gin-gonic/gin"
+	"github.com/pkg/errors"
+	"github.com/tikv/pd/pkg/errs"
+	"github.com/tikv/pd/pkg/mcs/utils"
+	"github.com/tikv/pd/pkg/storage/endpoint"
+	"github.com/tikv/pd/server"
+	"github.com/tikv/pd/server/apiv2/middlewares"
+)
+
+// RegisterTSOKeyspaceGroup registers keyspace group handlers to the server.
+func RegisterTSOKeyspaceGroup(r *gin.RouterGroup) {
+	router := r.Group("tso/keyspace-groups")
```

Comment:
Since we also use "tso/keyspace-groups" in the other packages, e.g., pkg/storage/endpoint, probaly we'd better define it as a const value in utils

Reply:
We use `tso/keyspace-groups` in API and `tso/keyspace_groups` in storage. slightly different